text/0045-nft-deposits-asset-hub.md

```diff
@@ -0,0 +1,246 @@
+creating an NFT collection, minting an individual NFT, and lowering its corresponding metadata and
+attribute deposits. The objective is to lower the barrier to entry for NFT creators, fostering a
+more inclusive and vibrant ecosystem while maintaining network integrity and preventing spam.
+
+## Motivation
+
+The current deposit of 10 DOT for collection creation (along with 0.01 DOT for item deposit and 0.2
+DOT for metadata and attribute deposits) on the Polkadot Asset Hub and 0.1 KSM on Kusama Asset Hub
+presents a significant financial barrier for many NFT creators. By lowering the deposit
+requirements, we aim to encourage more NFT creators to participate in the Polkadot NFT ecosystem,
+thereby enriching the diversity and vibrancy of the community and its offerings.
+
+The initial introduction of a 10 DOT deposit was an arbitrary starting point that does not consider
+the actual storage footprint of an NFT collection. This proposal aims to adjust the deposit first to
+a value based on the `deposit` function, which calculates a deposit based on the number of keys
+introduced to storage and the size of corresponding values stored.
+
+Further, it suggests a direction for a future of calculating deposits variably based on adoption
+and/or market conditions. There is a discussion on tradeoffs of setting deposits too high or too
+low.
+
+### Requirements
+
+- Deposits SHOULD be derived from `deposit` function, adjusted by correspoding pricing mechansim.
+
+## Stakeholders
+
+- **NFT Creators**: Primary beneficiaries of the proposed change, particularly those who found the
+  current deposit requirements prohibitive.
+- **NFT Platforms**: As the facilitator of artists' relations, NFT marketplaces have a vested
+  interest in onboarding new users and making their platforms more accessible.
+- **dApp Developers**: Making the blockspace more accessible will encourage developers to create and
+  build unique dApps in the Polkadot ecosystem.
+- **Polkadot Community**: Stands to benefit from an influx of artists, creators, and diverse NFT
+  collections, enhancing the overall ecosystem.
+
+Previous discussions have been held within the [Polkadot
+Forum](https://forum.polkadot.network/t/polkadot-assethub-high-nft-collection-deposit/4262), with
+artists expressing their concerns about the deposit amounts.
+
+## Explanation
+
+This RFC proposes a revision of the deposit constants in the configuration of the NFTs pallet on the
+Polkadot Asset Hub. The new deposit amounts would be determined by a standard deposit formula.
+
+As of v1.1.1, the Collection Deposit is 10 DOT and the Item Deposit is 0.01 DOT (see
+[here](https://github.com/polkadot-fellows/runtimes/blob/v1.1.1/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L687)).
+
+Based on the storage footprint of these items, this RFC proposes changing them to:
+
+```rust
+pub const NftsCollectionDeposit: Balance = system_para_deposit(1, 130);
```

Comment:
If there are two items then we should probably be consistent. It should still be a large reduction (from 10 to 0.4 instead of 0.2).

Broadly speaking though, and w/r/t the idea of finding an "optimal" deposit, it can never be perfect. As long as you can't start adding more storage items for free (i.e. deposit is directly proportional to storage items added) then it's well-designed. Factors of 2x are just not significant (and it's a bit arrogant to think we can get it so precise). I think probably 10x is the best we can hope for with these parameters.